### ISSUE TITLE
ie not rendering mailto correct

### ISSUE THREAD

**Issue by cemik** [skydiver/october-plugin-ssbuttons#6]
I'm using the simple sharing buttons and the mailto-share-button is rendered fine in all browsers but ie, where I only get 
```html
<a title="Email" href="mailto:?subject=___title___&amp;body=___title___:%20___url___" target="_blank"><img alt="Email" src="/plugins/martin/ssbuttons/assets/imgs/flat_web_icon_set_color/email.png"></a>
```
instead of 
```html
<a href="mailto:?subject=October%20CMS%20-%20Vanilla&amp;body=October%20CMS%20-%20Vanilla:%20http://october.cms/" title="Email" target="_blank"><img src="/plugins/martin/ssbuttons/assets/imgs/flat_web_icon_set_color/email.png" alt="Email"></a>
```

**Comment by sean-ciaschi** [comment]
From Issue #7, changing  line 34 to `$url = ($this->properties['js']) ? 'url' : url($this->currentPageUrl()); ` in SSButtons.php, SSButtonsNB.php, SSButtonsSSB.php had fixed that issue. I tested it in all browsers. If you have any problems with it, let me know and ill look into it for you

**Comment by skydiver** [comment]
If this issue is related to #7 then was fixed on version 1.3.0

I don't have access to IE so if anybody can check the fix will be great.

Thanks.-

**Comment by cemik** [comment]
Sorry, not fixed in 1.3.0 when "Use JS for Title & URL" is enabled. Same issue in IE occurs.

**Comment by sean-ciaschi** [comment]
I'll look into it now. What version of IE are you using? Edge?

**Comment by cemik** [comment]
I can only test on ie 11 on my local dev machine, but when I disovered the issue I believe it occured both in ie 11 and edge on the production site.

**Comment by sean-ciaschi** [comment]
I believe i found it. IE doesnt support forEach loops so instead the code inside the ssbuttons.js should be
```
var title = document.title;
var url   = document.location.href;
var links = document.querySelectorAll('.SocialSharingButtons a');

window.onload = function()
{
    for(var i = 0; i < links.length; i++)
    {
        links[i].href = links[i].href.replace(/___title___/g, encodeURI(title));
        links[i].href = links[i].href.replace(/___url___/g  , encodeURI(url));
    }
};
```

it seemed to work when i used it so let me know if that works for you.

**Comment by cemik** [comment]
Tested it IE 11 and Edge and it works perfect.  :)

**Comment by sean-ciaschi** [comment]
@cemik Awesome, let me know if you have any more issues 😄

**Comment by skydiver** [comment]
This was fixed on v1.3.0.1

Thanks @sean-ciaschi :thumbsup:
